phpBB/styles/prosilver/theme/cp.css

```diff
@@ -91,6 +91,32 @@ ul.cplist {
 	margin: 0;
 }
 
+.left-inner {
+	font-size: 13px;
+	position: relative;
+	padding: 5px;
+}
+
+.time-date {
+	font-size: 12px;
+	position: absolute;
+	right: 5px;
+}
+
+.message-row {
+	color: #888f98;
+	padding: 10px;
+	list-style-type: none;
+}
+
+.left-panel {
+	width: 35%;
+}
+
+.msg-unread {
+	box-shadow: 5px 0 0 0 #f7d05a inset;
```

Comment:
The color here should move to colors.css

Reply:
It appears box-shadow properties cannot be targetted separately. Should I move it entirely to colors.css

Reply:
I would say yes, since the only point is really about an aesthetic, rather than positional or formatting.